helpers/log_folder.sh

```diff
@@ -47,4 +47,24 @@ log_folder()
         ;;
     esac
   fi
+
+  readarray -t D_LOG_FILES < <( find . \( -path ./external -o -path ./config \) -prune -false -o \( -name "*.txt" -o -name "*.log" \) -exec md5sum {} \; 2>/dev/null | sort -u -k1,1 | cut -d\  -f3 )
```

Comment:
could we move this to the prepare helper file? It probably fits there quite fine

Reply:
Done.